crates/bevy_ecs/examples/events.rs

```diff
@@ -0,0 +1,48 @@
+use bevy_ecs::{event::Events, prelude::*};
+
+// This is our event
+#[derive(Debug)]
+struct MyEvent {
+    pub message: String,
+    pub random_value: f32,
+}
+
+fn main() {
+    // Create a world and add the event as a resource
+    let mut world = World::new();
+    world.insert_resource(Events::<MyEvent>::default());
+
+    // Create a schedule and a stage
+    let mut schedule = Schedule::default();
+    let mut update = SystemStage::parallel();
+
+    // Add the event managing system to the stage
+    update.add_system(Events::<MyEvent>::update_system.system());
+
+    // Add systems sending and receiving events
+    update.add_system(sending_system.system());
+    update.add_system(receiving_system.system());
```

Comment:
We should probably add a dependency here. Otherwise we might receive first, resulting in a sent event not being processed until the next frame.